### ISSUE TITLE
Investigate how suitable DIMPACT model is for use in CO2.js

### ISSUE THREAD

**Issue by mrchrisadams** [thegreenwebfoundation/co2.js#141]
We support a number of models in CO2.js right now, because different models carry different assumptions, and depending on the activity you are trying to measure, one set of assumptions might model it better than others.

With this in mind, it seemed worth making an explicit issue to see how DIMPACT might fit into our work.

I think might be designed for understanding the impacts of streaming more than web apps, but either way it seems worth having a look into

https://dimpact.org/news

**Comment by mrchrisadams** [comment]
Some quick notes, after reading the Methodology Statement on the publications page of the DIMPACT website:

https://dimpact.org/publications

The DIMPACT model that was shared in October 2022, and used to make calculations is less complicated than I thought. 

The model is actually three models, or rather one model, with three 'modules' which tailor it slightly for:

1. video streaming (see [the carbon trust DIMPACT report](https://www.carbontrust.com/our-work-and-impact/guides-reports-and-tools/carbon-impact-of-video-streaming) for more)
2. digital publishing
3. online banner advertising (this one is referred to, but there is nothing online that I could see)

The second use case, digital publishing is probably the one that can be compared to models like the SWD model, the default in CO2.js.

I've summarised the main changes I see from SWD:

Component | SWD | DIMPACT
------|------------ | ------------
Lifecycle | Included, scales with data transfer | Not included
Server side | Included, scales with data transfer | included, does not scale with transfer, as based on primary number from supplier, except for CDNS
Network | Included, scales with data transfer | Included, scales with data transfer 
End user | Included, scales with data transfer | Included, scales based on time used by the end user device
CPE (local routers) | Rolled into Network segment  | Included, does not change and is assumed to be on 24/7

There is an expert panel section at the end, comprised of a few commonly cited names in this field (Jonathan Koomey, George Kamiya, Daniel Schien, Eric Masanet, Arman Shehabi, Jens Malmodin) - they basically say that they understand that network transfer based approaches have their issues, and as a result, they can't fully agree with the DIMPACT approach.

There's no better model they point to in the wild that can be used, and they acknowledge that this research has highlighted a need to develop better models.

So basically, just like the SWD model, it's a model, and as we all know, _all models are wrong, but some are useful_.

It relies on a few more inputs than SWD or 1Byte, but doesn't go into as much detail as the greenframe model, as it's not intended to be used the same way.

But on the bright side, it does give some guidance on how to fit it into GHG reporting guidelines.

The report is about 40 pages long including appendixes and it's quite readable.
